holoviews/plotting/bokeh/callbacks.py

```diff
@@ -152,39 +164,84 @@ def __init__(self, plot, streams, source, **params):
         self.streams = streams
         self.comm = self._comm_type(plot, on_msg=self.on_msg)
         self.source = source
+        self.handle_ids = defaultdict(list)
 
 
     def initialize(self):
         plots = [self.plot]
         if self.plot.subplots:
             plots += list(self.plot.subplots.values())
 
```

Comment:
Looks like the code below could be made into a small ``get_handles`` (or ``get_plot_handles``) method. Alternatively, it could be a function in utils so the code below could be:

``self.handle_ids.update(util.get_handles(plots))``

Which I think is clearer.